apps/parsley/src/components/DetailsMenu/DetailsMenuCard/styles.tsx

```diff
@@ -27,7 +27,6 @@ interface DetailsLabelProps {
 const DetailsLabel: React.FC<DetailsLabelProps> = ({ children, label }) => (
   <LabelWrapper>
     <DetailName>{children}</DetailName>
-    {/* @ts-expect-error https://jira.mongodb.org/browse/LG-3797 */}
```

Comment:
It makes me sad that typescript didn't catch this before :(

I wonder if there was something wrong with how we declared `@ts-expect-error`?